samples/matter/lock/src/lit/zap-generated/IMClusterCommandHandler.cpp

```diff
@@ -450,73 +452,8 @@ namespace app
 
 		} // namespace GroupKeyManagement
 
-		namespace IcdManagement
```

Comment:
I think you accidentally removed commands handling from the ICD Management cluster. Did you re-generated `lit/zap-generated` files using `lock.zap` file? This one is for SIT, for LIT we have a separate one - `lock-lit.zap`.